### PR TITLE
HPCC-16945 Cookie detection for session mgmt

### PR DESCRIPTION
**Pull request by miguelvazq**
Add a cookie detection and relay message to user if cookies are disabled.

Signed-off by: Miguel Vazquez <miguel.vazquez@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [ ] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [x] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [x] I have tested my changes in multiple modern browers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by miguelvazq** [comment]
@GordonSmith Please review. @rpastrana Please review translations @kunalaswani Verify your testings @RussWhitehead, @wangkx  FYI

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 1]
```diff
@@ -439,6 +439,7 @@ define(
     PlaceholderFirstName: "John",
     PlaceholderLastName: "Smith",
     Playground: "Patio de juegos",
+    PleaseEnableCookies: "ECL Watch requiere cookies habilitado para continuar.",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: @miguelvazq  Slight reward:
"ECL Watch requiere que las cookies esten habilitadas para poder continuar."

**Comment by kunalaswani** [comment]
Tested across IE, Firefox, and Chrome to ensure compatibility, working properly.

**Review by GordonSmith — APPROVED** [review]
Looks good.

**Pull request merged**